src/api/api.ts

```diff
@@ -0,0 +1,7 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Incorrect name of file. In this file you create methods for working with the user and it should be named accordingly.